pkgs/development/perl-modules/strip-nondeterminism/default.nix

```diff
@@ -36,5 +36,6 @@ buildPerlPackage rec {
     homepage = "https://reproducible-builds.org/";
     license = licenses.gpl3;
     maintainers = with maintainers; [ pSub ];
+    mainProgram = "strip-nondeterminism";
```

Comment:
That shouldn't be necessary.

Reply:
Unfortunately `buildPerlPackages` seems to mess with `pname`, so `nix run` basically doesn't work on anything in `perlPackages` by default :(

```console
~/C/nixpkgs on perlPackages-mainProgram
❯ nix eval -f . perlPackages.strip-nondeterminism.pname
"perl5.34.1-strip-nondeterminism"
```